File: DigitalSolutions.TestApp.WebAPI/DigitalSolutions.TestApp.WebAPI/DTO/AccountDTO.cs
using DigitalSolutions.TestApp.WebAPI.DataBase.DBContext;

namespace DigitalSolutions.TestApp.WebAPI.DTO
{
    public class AccountDTO
    {
        public string? createTime { get; set; }
        public string? changeTime { get; set; }
        public int accountNumber { get; set; }
        public string? accountStatus { get; set; }
        public long balance { get; set; }
        public string? paymentMethod { get; set; }
    }
}

File: DigitalSolutions-TestApp-UI/src/app/services/AccountDTO.ts
import { Injectable } from "@angular/core";

@Injectable()
export class AccountDTO
{
    createTime : string = "2000-01-01T00:00:00.0000000";
    changeTime : string = "2000-01-01T00:00:00.0000000";
    accountNumber : number = 0;
    accountStatus : string = "0";
    balance : number = 0;
    paymentMethod : string = "0";
}